<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComprayVenta</class>
 <widget class="QWidget" name="ComprayVenta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ventana_comprayventa">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>1031</width>
     <height>811</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Recursos/images.qrc">:/comprayventadesign.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="id_registroventas">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Registro de venta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_busquedaventas">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Busqueda de venta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_registrocompras">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Registro de compras</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_guiacompras">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gu√≠a de compras</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_regresardecomprayventa">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Recursos/regreso.png</normaloff>../Recursos/regreso.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Recursos/images.qrc"/>
 </resources>
 <connections/>
</ui>
